src/wallet.ts

```diff
@@ -21,7 +21,10 @@ interface TypedMessageParams extends MessageParams {
   version: string;
 }
 interface WalletMiddlewareOptions {
-  getAccounts: (req: JsonRpcRequest<unknown>) => Promise<string[]>;
+  getAccounts: (
+    req: JsonRpcRequest<unknown>,
+    suppressUnauthorized?: boolean,
```

Comment:
Maybe instead of a positional argument this should be in an object? `{ suppressUnauthorized = false }: { suppressUnauthorized?: boolean }`. That way when calling `getAccounts` with `false` it's a little more clear what the `false` is doing. Also see https://github.com/MetaMask/metamask-extension/pull/14126#discussion_r853335734

Reply:
Oh I see, Erik already commented on this. Well, up to you, then.